gpMgmt/bin/gppylib/gpcatalog.py

```diff
@@ -21,8 +21,32 @@
 class GPCatalogException(Exception):
     pass
 
-# Hard coded since "coordinator only" is not defined in the catalog
-COORDINATOR_ONLY_TABLES = [
+# The following lists capture "coordinator only" tables. They are hard-coded
+# since there is no notion of "coordinator only" defined in the catalog.
+# Depending on whether a relation is mapped or not, we have 2 sub-lists.
+# See RelationIsMapped().
+
+COORDINATOR_ONLY_TABLES_MAPPED = [
+    'pg_auth_time_constraint',
+    'pg_shdescription',
+    'gp_configuration_history',
+    'gp_segment_configuration',
+    'pg_stat_last_shoperation',
+    'gp_matview_aux',
+    'gp_matview_tables',
```

Comment:
what about table `gp_matview_dependency`?

Reply:
IDK, it was not cleared before this PR, so I decided to leave it as it is. Actually, list looks suspicious to me, see comments about "gp_segment_configuration".

Reply:
> what about table `gp_matview_dependency`?

i dont see such table...

db1=# \d+ gp_matview_dependency
Did not find any relation named "gp_matview_dependency".